### PR TITLE
Fix actions positioning

### PR DESCRIPTION
**Pull request by juliushaertl**
- Limit height of the NcActions popover
- Fix placement of the NCAppNavigationItem actions menu

### CODE REVIEW

File: src/components/NcAppNavigationItem/NcAppNavigationItem.vue [hunk 1]
```diff
@@ -175,8 +175,7 @@ Just set the `pinned` prop.
 				</div>
 				<NcActions v-show="displayActionsOnHoverFocus || forceDisplayActions"
 					ref="actions"
-					menu-align="right"
-					:container="'#' + id"
+					:boundaries-element="'#content-vue'"
```
Comment (raimund-schluessler) [inline src/components/NcAppNavigationItem/NcAppNavigationItem.vue:178]:
```suggestion
					boundaries-element="#content-vue"
```

**Review by ChristophWurst — APPROVED** [review]

**Comment by raimund-schluessler** [comment]
Will this solve #3146?

File: src/components/NcAppNavigationItem/NcAppNavigationItem.vue [hunk 2]
```diff
@@ -391,7 +390,7 @@ export default {
 		 */
 		menuPlacement: {
 			type: String,
-			default: 'bottom',
+			default: 'auto',
```
Comment (raimund-schluessler) [inline src/components/NcAppNavigationItem/NcAppNavigationItem.vue:393]: Can we keep this at `bottom`? It works the equally well and does not change the old behaviour. Or was there a specific reason / design decision for this change?
```suggestion
			default: 'bottom',
```

File: src/components/NcAppNavigationItem/NcAppNavigationItem.vue [hunk 3]
```diff
@@ -175,8 +175,7 @@ Just set the `pinned` prop.
 				</div>
 				<NcActions v-show="displayActionsOnHoverFocus || forceDisplayActions"
 					ref="actions"
-					menu-align="right"
-					:container="'#' + id"
+					boundaries-element="#content-vue"
```
Comment (raimund-schluessler) [inline src/components/NcAppNavigationItem/NcAppNavigationItem.vue:178]: If we set the container to `#app-navigation-vue`, the hidden overflow issue in #3146 is fixed as well, but additionally, the Actions menu is not jumping/moving when the body is scrolled.
```suggestion
					container="#app-navigation-vue"
					boundaries-element="#content-vue"
```

**Review by raimund-schluessler — CHANGES_REQUESTED** [review]
I can confirm that this fixes #3146.
However, I would propose to set the `container` properly, see https://github.com/nextcloud/nextcloud-vue/pull/3157/files#r961861481

**Comment by GretaD** [comment]
@raimund-schluessler Julius is on holiday, do you want to take over and deliver the(your) comments :P. I can also rebase and change the stuff as you proposed

**Comment by raimund-schluessler** [comment]
> @raimund-schluessler Julius is on holiday, do you want to take over and deliver the(your) comments :P. I can also rebase and change the stuff as you proposed

Sure, I can do that. I just didn't want to force my changes, as there might be specific reasons for the changes. But I guess we can still adjust it before a stable 6.0.0. release if necessary.

**Review by raimund-schluessler — APPROVED** [review]

**Pull request merged**